ts/screens/messages/MessageRouterScreen.tsx

```diff
@@ -11,7 +11,7 @@ import { navigateToEuCovidCertificateDetailScreen } from "../../features/euCovid
 import { EUCovidCertificateAuthCode } from "../../features/euCovidCert/types/EUCovidCertificate";
 import I18n from "../../i18n";
 import { mixpanelTrack } from "../../mixpanel";
-import { loadMessages } from "../../store/actions/messages";
+import { DEPRECATED_loadMessages as loadMessages } from "../../store/actions/messages";
```

Comment:
Here we have a problem. I don't know if you are aware of this, I don't.
This "router" component just loads all messages to get their details. This is an issue since it executes the same path of the previous API:

- tap in message item
- navigate to message router
- call the getMessages API (no pagination)
- call the message detail API foreach message

![Schermata 2021-11-09 alle 18 00 00](https://user-images.githubusercontent.com/822471/140970198-4431d87b-c9df-45de-8ead-5e60dc55387e.png)

### Here a video
https://user-images.githubusercontent.com/822471/140970250-255cdeee-bf67-4e0f-94fd-c7b23c25de42.mov

It is for sure a problem about performance but also for those users that have > 100 messages
I believe we should fix this behavior in a dedicated story by tading advantage of the feature flag to create a new dedicate message router. What do you think?